pkgs/development/libraries/clanlib/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, libGL
+, libpng
+, pkg-config
+, xorg
+, file
+, freetype
+, fontconfig
+, xlibsWrapper
+, alsaLib
+, libXrender
+}:
+
+stdenv.mkDerivation rec {
+  pname = "clanlib";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    repo = "ClanLib";
+    owner = "sphair";
+    rev = "v${version}";
+    sha256 = "sha256-SVsLWcTP+PCIGDWLkadMpJPj4coLK9dJrW4sc2+HotE=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    autoreconfHook
+  ];
+  buildInputs = [
+    libGL
+    libpng
+    xorg.xorgproto
+    freetype
+    fontconfig
+    xlibsWrapper
+    alsaLib
+    libXrender
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/sphair/ClanLib";
+    description = "A cross platform toolkit library with a primary focus on game creation";
+    license = licenses.mit;
+    maintainers = with maintainers; [ nixinator ];
+    platforms = platforms.x86_64-linux;
```

Comment:
```suggestion
    platforms = [ "x86_64-linux" ];
```

Reply:
A lot of packages seem to use `platforms.x86_64-linux`, what is the benefit of setting it as a string? Do you have any examples of `platforms.x86_64-linux` causing trouble? I'm quite interested in the semantics of this :smile:

Reply:
`platforms.x86_64-linux` doesn't exist, see the failed `ofborg-eval` check on this very PR.

> while evaluating the attribute 'meta.platforms'
> attribute 'x86_64-linux' missing

Reply:
Thanks @OPNA2608 I was conflating this with other things that exist. Like how in my flake I have `packages.x86_64-linux`, easy enough mistake to make :) 

https://github.com/MatthewCroughan/nixcfg/blob/3ce32c7cc479f11d01ffcf062741df6631b11d4f/flake.nix#L29